live-tests/tests/test_nexus_add_remove.rs

```diff
@@ -0,0 +1,229 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+mod common;
+
+use anyhow::Context;
+use assert_matches::assert_matches;
+use common::reconfigurator::blueprint_edit_current_target;
+use common::LiveTestContext;
+use futures::TryStreamExt;
+use live_tests_macros::live_test;
+use nexus_client::types::Saga;
+use nexus_client::types::SagaState;
+use nexus_reconfigurator_planning::blueprint_builder::BlueprintBuilder;
+use nexus_reconfigurator_planning::blueprint_builder::EnsureMultiple;
+use nexus_reconfigurator_preparation::PlanningInputFromDb;
+use nexus_sled_agent_shared::inventory::ZoneKind;
+use nexus_types::deployment::SledFilter;
+use omicron_common::address::NEXUS_INTERNAL_PORT;
+use omicron_test_utils::dev::poll::wait_for_condition;
+use omicron_test_utils::dev::poll::CondCheckError;
+use slog::{debug, info};
+use std::net::SocketAddrV6;
+use std::time::Duration;
+
+// TODO-coverage This test could check other stuff:
+//
+// - that after adding:
+//   - the new Nexus appears in external DNS
+//   - we can _use_ the new Nexus from the outside
```

Comment:
Possibly not today.  I suspect figuring this out will be worth the investment.  Maybe we can use the techport interface for the external API?

Reply:
We could, but I'm not sure it gains us much. If we're in the switch zone we can either connect to the TCP proxy wicketd runs or directly to the "Nexus external API exposed on the underlay" that wicketd forwards to. But in either case, it wouldn't really confirm external connectivity, because it's not actually using the external IP. It does confirm the new Nexus would be reachable via the techport, which is something though.